tests/main.spec.js

```diff
@@ -31,7 +31,7 @@ describe('bsCustomInputFile', function () {
     it('should add bsCustomFileInput property', function () {
       bsCustomFileInput.init()
 
-      expect(input.bsCustomFileInput).to.not.undefined
+      expect(input.bsCustomFileInput).not.undefined
```

Comment:
Shouldn't this include `be`?

Reply:
Hmm I think it should works too 🤔 but I don't which one is better

Reply:
Your call :)